We cannot run the no_jupyter tests because we always have the jupyter extras
installed.

--- black-24.8.0/tox.ini.orig
+++ black-24.8.0/tox.ini
@@ -19,9 +19,6 @@
 ; local runs can stay parallelized since they aren't rolling the dice so many times as like on CI
 commands =
     coverage erase
-    pytest tests --run-optional no_jupyter \
-        !ci: --numprocesses auto \
-        --cov {posargs}
     pytest tests --run-optional jupyter \
         -m jupyter \
         !ci: --numprocesses auto \
